generatedTypes/components/card/index.d.ts

```diff
@@ -83,7 +83,7 @@ declare const _default: React.ComponentClass<ViewProps & TouchableOpacityProps &
     /**
      * action for when pressing the card
      */
-    onPress?: (() => void) | undefined;
+    onPress?: ((props?: any) => void) | undefined;
```

Comment:
Is typescript still make us use `| undefined` although `onPress` is an optional?

Reply:
You are looking at the generatedTypes file, this is how it generates the d.ts files apparently. 
The source code doesn't have it